kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTaskTest.java

```diff
@@ -321,6 +379,25 @@ public static InsertAllRequest buildExpectedInsertAllRequest(
   /**
    * Utility method for spoofing SinkRecords that should be passed to SinkTask.put()
    * @param topic The topic of the record.
+   * @param value The content of the record.
+   * @param timestampType The type of timestamp embedded in the message
+   * @param timestamp The timestamp in milliseconds
+   * @return The spoofed SinkRecord.
+   */
+  public static SinkRecord spoofSinkRecord(String topic, String value, TimestampType timestampType, Long timestamp) {
```

Comment:
Lets make previous spoofSinkRecord be made to call this one, for more ensured consistency